### DISCUSSION TITLE
Why nuxt build doesn't fail when used components are not imported

### DISCUSSION THREAD

**Discussion by brachkow** [nuxt/nuxt#9548] [Questions (Nuxt 2)]
Why does `nuxt build` doesn't fail when used components are not imported? It leads to situations when build succeeds but inside app component throws an error to console.

P.S. `components: false` is set!

**Comment by kissu** [comment]
Hey, please edit your question and reply to my answer on StackOverflow if you want to move forward on this issue! 😉
